src/reference/00-Getting-Started/01-Setup/d.md

```diff
@@ -0,0 +1,126 @@
+---
+out: Installing-sbt-native-client.html
+---
+
+Installing sbtn
+-----------------------
+
+sbtn is a client that is able to connect to a running sbt server to issue
+commands. Because sbt builds may take some time to load, using sbtn can
+significantly improve the experience of using sbt from the command line. Binary
+distributions are available for 64 bit Linux, MacOS and Windows. sbtn includes
+custom tab completions for bash, fish, powershell and zsh so that you can have
+an experience similar to the embedded sbt shell directly from the command line.
+
+### Installing using built in sbt wizard
+
+If you have previously installed sbt $app_version$ or greater, then you can install sbtn
+by running
+
+```
+sbt installSbtn
+```
+
+or simply run `installSbtn` in an interactive sbt shell session. The wizard will
+download and install the native binary into a subdirectory of the sbt global
+directory, typically found in `~/.sbt/1.0/bin` on most systems. In addition to
+downloading the native binary, you will be prompted to setup sbtn for your
+shell. The wizard is able to configure your shell environment for bash, fish,
+powershell or zsh by updating the standard configuration files. The wizard can
+update your shell configuration to add `~/sbt/1.0/bin` to the shell
+[PATH](https://kb.iu.edu/d/acar) variable. It also add the commands necessary to
+enable the custom tab completions.
+
+### Installing using Coursier
+
+[Coursier](https://get-coursier.io) can be used to install to sbtn using the
+`cs` command line tool. Simply run:
+
+```
+cs install sbtn
+```
+
+### Manual installation
+
+Manual installation of sbtn requires that you download the native binary for
+your operating system, optionally download a shell completion script to add custom
+tab completions to your shell of choice and optionally update the configuration
+file for your shell so that sbtn and its tab completions are available from the
+shell prompt. The completion scripts are rarely updated so these will not typically need to be
+updated along with sbt. The sbtn binary, on the other hand, should ideally be
+updated with each new sbt release.
+
+
+Binary releases of sbtn are available at
+[sbtn-dist](https://github.com/sbt/sbtn-dist/tags). The latest platform specific
+releases can be found at:
+
+* Linux <https://github.com/sbt/sbtn-dist/releases/download/v$app_version$/sbtn-x86_64-pc-linux-$app_version$.tar.gz>
+
+* MacOS <https://github.com/sbt/sbtn-dist/releases/download/v$app_version$/sbtn-x86_64-apple-darwin-$app_version$.tar.gz>
+
+* Windows <https://github.com/sbt/sbtn-dist/releases/download/v$app_version$/sbtn-x86_64-pc-win32-$app_version$.zip>
```

Comment:
sbtn version doesn't always correspond with sbt version, since I was hoping that we wouldn't have to update it once it's stable. For example sbt 1.4.1 uses sbtn 1.4.0, I think.

Reply:
I wondered about that. The reason I put the app_version in the url is because I wanted the download urls to be available and reflect the latest code. I would suggest that until sbtn is truly stable that we keep releasing sbtn versions along with sbt versions even if nothing has changed. It makes it easier to think about versions if they are kept in lockstep both for documentation and troubleshooting. I can think of a few alternatives though:
1. Provide urls that always redirect to the latest version of sbtn
2. Add an` sbtn_version` to the project templates and make sure that we update it along with app_version whenever making a new release (independent of sbtn installation instructions, it seems like bumping app_version should be added to the release process).

I would prefer Option 1 to what I did in this PR but I don't know how to implement it.